filebeat/fileset/fileset_test.go

```diff
@@ -183,17 +183,17 @@ func TestGetProspectorConfigNginxOverrides(t *testing.T) {
 	assert.True(t, cfg.HasField("pipeline"))
 	pipelineID, err := cfg.String("pipeline", -1)
 	assert.NoError(t, err)
-	assert.Equal(t, "nginx-access-with_plugins", pipelineID)
+	assert.Equal(t, "nginx-access-with_plugins-5.2.0", pipelineID)
```

Comment:
I would call the pipeline `filebeat-5.2.0-nginx-access-with_plugins`. This brings the following benefits:

* No conflict with potential pipelines in the future from other beats
* Assuming some can query for pipelines, to get / remove all filebeat pipelines `filebeat-*` can be used, to remove all `5.2` pipelines because everyting is using 5.3, `filebeat-5.2.*` can be used.